### PR TITLE
Custom/18970 remove some redundant config values

### PR DESCRIPTION
**Pull request by devProdigy**
## Description

remove some redundant config values and related to it code.
TODO:
- clean PROD and DEV configs accordingly before deploy

[Azure DevOps Link](https://dev.azure.com/EquinorASA/OMNIA%20Prevent/_workitems/edit/18970)

## Type of change

Mark the relevant options (`[x]` will mark checkbox as checked).
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [X] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [X] Documentation content changes
- [ ] Tests
- [ ] Other

# Checklist:

- [X] I have made corresponding changes to the documentation / README (if needed)
- [ ] I have added tests that prove my fix is effective or that my feature works (if needed)

### CODE REVIEW

File: app/latigo/scheduler/__init__.py [hunk 1]
```diff
@@ -200,24 +184,12 @@ def on_time(self):
         except KeyboardInterrupt:
             logger.info("Keyboard abort triggered, shutting down")
             return True
-        except Exception:
-            logger.exception(f"Error occurred in scheduler")
+        except Exception as e:
+            logger.exception(f"Error occurred in scheduler: {e}")
```
Comment (o-fedorov) [inline app/latigo/scheduler/__init__.py:188]: There is no need to pass an exception explicitly, it is to be appended when the logs are printed out.

File: app/latigo/scheduler/__init__.py [hunk 2]
```diff
@@ -226,15 +198,8 @@ def run(self):
 
         while not done:
             logger.info(
-                f"Next prediction will occur at {self.continuous_prediction_timer.closest_start_time()} (in {human_delta(self.continuous_prediction_timer.time_left())})"
+                f"Next prediction will occur at {self.continuous_prediction_timer.closest_start_time()} "
+                f"(in {human_delta(self.continuous_prediction_timer.time_left())})"
```
Comment (o-fedorov) [inline app/latigo/scheduler/__init__.py:202]:
```suggestion
                "Next prediction will occur at %s (in %s)", 
                self.continuous_prediction_timer.closest_start_time(),
                human_delta(self.continuous_prediction_timer.time_left())
```

Strings substitution with `%` is a recommended way for logging.

Comment (devProdigy) [reply app/latigo/scheduler/__init__.py]: done

File: app/latigo/scheduler/__init__.py (hunk 1 above)
Comment (devProdigy) [reply app/latigo/scheduler/__init__.py]: done

**Pull request merged**